src/openklant/components/klantinteracties/api/tests/test_actoren.py

```diff
@@ -344,7 +355,12 @@ def test_destroy_geatomatiseerde_actor(self):
         self.assertEqual(data["count"], 0)
 
 
-class MedewerkerTests(JWTAuthMixin, APITestCase):
+class MedewerkerTests(APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.token_auth = TokenAuthFactory.create()
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.token_auth.token)
```

Comment:
Please make this part of the `APITestCase` Maybe even in `setUpClass` and remove it in all tests.